community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/variables.tf

```diff
@@ -35,7 +35,7 @@ variable "enable_debug_logging" {
 
 variable "extra_logging_flags" {
```

Comment:
Should we just remove this or update this extra_loggin_flags --> trace_api ?

Reply:
This is out of scope, let's not change interfaces